custom_model_runner/datarobot_drum/drum/adapters/model_adapters/python_model_adapter.py

```diff
@@ -548,6 +550,9 @@ def _predict_legacy_drum(self, data, model, **kwargs):
                     exc, "Model 'score' hook failed to make predictions."
                 )
             self._validate_data(predictions_df, CustomHooks.SCORE)
+            predictions_df, extra_df = self._split_to_predictions_and_extra_df(
```

Comment:
this is currently _only_ implemented in python, correct?  will it get implemented in other languages?

Reply:
Correct, the plan is to add implement it also in other languages.